uwuno/pom.xml

```diff
@@ -59,7 +59,13 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>3.8.1</version>
 		</dependency>
-	</dependencies>
+	
```

Comment:
Do we need a specific function only available in 3.9, 3.8.1 is included right above this.

Reply:
Nope LOL